### ISSUE TITLE
wrapArguments doesn't work

### ISSUE THREAD

**Issue by subdan** [nicklockwood/SwiftFormat#559]
I have this code:
```
protocol Reporter {
    func reportBegin(
        name: String, deviceId: String,
        connection: PhoneConnection,
        completion: @escaping (Error?) -> Void)
}
```
SwiftFormat unable to wrap `deviceId` parameter.
Actual output:
```
protocol Reporter {
    func reportBegin(
        name: String, deviceId: String,
        connection: PhoneConnection,
        completion: @escaping (Error?) -> Void)
}
```
Expected output:
```
protocol Reporter {
    func reportBegin(
        name: String,
        deviceId: String,
        connection: PhoneConnection,
        completion: @escaping (Error?) -> Void)
}
```

**Comment by AlexisQapa** [comment]
The current behaviour is to wrap each time the args would be longer than the line limit, so it keep two short args on the same line, so it works as intended I think. Thought it's true that most conventions I came across would either wrap none or all args. I think this second behaviour should be the default but maybe someone would like the current one

**Comment by nicklockwood** [comment]
@subdan yes, @AlexisQapa is correct. I think it may be worth adding another option to always wrap though.

**Comment by AlexisQapa** [comment]
As stated above I think the either wrap none or all would be better as the default option thought I don't know if this project have kinda feature stability as requirements

**Comment by nicklockwood** [comment]
I found that made a lot of undesirable changes in my own projects when I tried it, so I don't think it's a good choice as the default.

**Comment by 1oo7** [comment]
Yeah current behavior is not good.

**Comment by 1oo7** [comment]
I think regardless of `maxwidth`, if someone has manually wrapped a line, then SwiftFormat should enforce that the wrapping is done properly.

That's the default behavior when there is no `maxwidth` on the latest version so I'm not sure why it would stop working just because a `maxwidth` has been specified.

**Comment by Rhywun** [comment]
> most conventions I came across would either wrap none or all args. 

FWIW I was also expecting this option to place each argument on a separate line and was surprised at the actual behavior.

I hope an option can be added to allow this.
